script.js

```diff
@@ -0,0 +1,12 @@
+// Import Modules
+import getFormattedDateTime from './modules/date.js';
+import Library from './modules/library.js';
+import Navigation from './modules/navigation.js';
+
+document.querySelector('#date-time').innerHTML = getFormattedDateTime();
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be great if you update your date and time dynamically after every minute because now you are just getting your date and time just once and it stayed on your html page as static content 😨 You can make it dynamic by using setInterval which I used below 👇 You can use that code to make your date and time dynamic to make your project look more professional 👍

```javascript
const timeDate = () => {
  setInterval(() => {
    document.querySelector('#date-time').innerHTML = getFormattedDateTime();
  }, 100);
}
timeDate();

```